system/mp/mpcomm.cpp

```diff
@@ -2312,12 +2320,14 @@ CMPChannel *CMPServer::lookup(const SocketEndpoint &endpoint)
 }
 
 
-CMPServer::CMPServer(unsigned __int64 _role, unsigned _port)
+CMPServer::CMPServer(unsigned __int64 _role, unsigned _port, bool _listen)
 {
     RTsalt=0xff;
     role = _role;
     port = 0;   // connectthread tells me what port it actually connected on
     checkclosed = false;
+    listen = _listen;
+
     connectthread = new CMPConnectThread(this, _port);
```

Comment:
Not new that this 'thread' object creates the port if _port ==0, but now that it might not even start..
I think should have a comment block above this line to help reader, saying it what semantics of _port == 0 mean and listen = false.

Reply:
Added comment.